### ISSUE TITLE
32 bit version 

### ISSUE THREAD

**Issue by Skywalker-gamefu** [boniatillo-com/PhaserEditor#13]
any plan to do 32 bit version of phaser editor ?

**Comment by boniatillo-com** [comment]
We take note, also for a win 32 bit version. Thanks.

**Comment by PhaserEditor2D** [comment]
Hi @nalcwap, no, the editor cannot run con android.

**Comment by nalcwap** [comment]
sorry my bad english, yes i know but not possible make the application for android?thanks

**Comment by PhaserEditor2D** [comment]
No, it is full dependent of desktop environments. In a future we should implement some web tools, but  for the moment it is just a dream.

**Comment by nalcwap** [comment]
ok thanks

**Comment by amadich** [comment]
> We take note, also for a win 32 bit version. Thanks.

When is the version released?

**Comment by PhaserEditor2D** [comment]
Hi @amadich there isn't a 32bit version available yet.

For now you can try running the editor in the cloud. For example, here: https://phasereditor2d.com/start/
Click on the Create Online link of any template.
